src/__tests__/validator.test.ts

```diff
@@ -15,6 +15,7 @@ describe('validator', () => {
         const { isValid, errors, errorsText } = validator(definition);
         const textErrors = errorsText();
         if (match[1] === 'valid') {
+          expect(errors ?? []).toStrictEqual([]);
```

Comment:
yes, can definitely remove them

this change offered better output when the expect failed. 

also thinking there should be separate tests that assert the specific error outputs. It's not uncommon to have an error storm with `oneOf` since the error includes all variants. 

It's probably sufficient to emit the instancePath to the error. This is another nice thing we're leveraging from AJV. If we traversed the graph ourselves, we'd need to build up the path to the errors via a stack or similar.